modules/virt-creating-a-custom-image-boot-source-web.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-creating-a-custom-disk-image-boot-source-web_{context}"]
+= Creating a custom boot source
+
+Sometimes you want to prepare a custom disk image, based on an existing disk image, for use as a boot source.
+
+This procedure guides you through:
+
+* Preparing a custom disk image.
+* Creating a boot source from the custom disk image.
+* Attaching the boot source to a custom template.
+
+.Procedure
+. In the {VirtProductName} console, click *Workloads* > *Virtualization* from the side menu.
+. Click the *Templates* tab.
+. Click the link in the *Source provider* column for the template you want to customize. A window displays, indicating that the template currently has a *defined* _source_.
+. In the window, click the *Customize source* link.
+. In the *About boot source customization* window, click *Continue* to proceed with customization, after reading the information provided about the boot source customization process.
+. On the *Prepare boot source customization* page, in the *Define new template* section:
+.. Select the *New template namespace* field and choose *PR default*.
+.. Enter the name of the custom template in the *New template name* field.
+.. Enter the name of the template provider in the *New template provider* field.
+.. Select the *New template support* field and choose the appropriate value, indicating support contacts for the custom template you create.
+.. Select the *New template flavor* field and choose the appropriate CPU and memory values for the custom image you create.
+. In the *Prepare boot source for customization* section, customize the `cloud-init` YAML script, if needed, to define login credentials. Otherwise, default credentials will be automatically generated for you.
+. Click *Start Customization*. The customization process begins and the *Preparing boot source customization* page displays, followed by the *Customize boot source* page. The *Customize boot source* page displays the output of the running script. When the script completes, your custom image is created.
+. In the *VNC console*, click *show password* in the *Guest login credentials* section. Your login credentials are displayed.
+. When the image is ready for login, sign in with the *VNC Console* by providing the user name and password displayed in the *Guest login credentials* section.
+. Verify the custom image works as expected. If it does, click *Boot source available*.
```

Comment:
it's `Make this boot source available` on this window.

Reply:
Done.